### ISSUE TITLE
Make it possible to hide comments by certain users

### ISSUE THREAD

**Issue by vlidholt** [newsvoice/newsvoice#188]
Make a personal block list, so that you don't need to see comments from certain users.

**Comment by JustMeSaying** [comment]
Doing a personal block list allows the retention of personal echo chambers, doesn't fit the mission

**Comment by frog712** [comment]
Agreed, this would defeat the whole purpose of Newsvoice but it raises the issue of spam. Obviosuly we cannot have users reporting spam either, because authoritarian ideologues would report everything they don't agree with.
I think the best option would be some kind of bot that looks for gibberish or repeated long comments by a single user, then marks that user for investigation by an admin, who can then hide/delete individual or all comments from that user.
